src/items/equipment/early_stage_adaptation_graft.json

```diff
@@ -28,40 +43,170 @@
     },
     "attuned": false,
     "bulk": "-",
+    "chatFlavor": "",
+    "container": {
+      "contents": [],
+      "includeContentsInWealthCalculation": true,
+      "isOpen": true,
+      "storage": []
+    },
     "critical": {
+      "effect": "",
       "parts": []
     },
     "damage": {
       "parts": []
     },
+    "damageNotes": "",
     "description": {
       "chat": "",
+      "gmnotes": "",
+      "short": "",
       "unidentified": "",
-      "value": "<p>Even the youngest barathus have control over their mutable bodies. A series of extendable plates, elastic filaments, and reflective layers are installed throughout your skin and arms to provide you similar benefits. Once every 1d4 rounds as a swift action, you can reshape part of your body and adjust your chemistry to gain one of the following qualities. The adaptation lasts until the beginning of your next turn.</p>\n<ul>\n<li>\n<p>You can add an additional amount of damage to melee attacks equal to your Strength modifier.</p>\n</li>\n<li>\n<p>You gain a +1 enhancement bonus to AC.</p>\n</li>\n<li>\n<p>You gain resistance 2 against a single energy type (acid, cold, electricity, fire, or sonic).</p>\n</li>\n<li>\n<p>Your reach is extended by 5 feet.</p>\n</li>\n</ul>"
+      "value": "<p>Even the youngest @UUID[Compendium.sfrpg.races.RXNBF1w6yXaB8P9O]{barathus} have control over their mutable bodies. A series of extendable plates, elastic filaments, and reflective layers are installed throughout your skin and arms to provide you similar benefits. Once every 1d4 rounds as a swift action, you can reshape part of your body and adjust your chemistry to gain one of the following qualities. The adaptation lasts until the beginning of your next turn.</p><ul><li><p>You can add an additional amount of damage to melee attacks equal to your Strength modifier.</p></li><li><p>You gain a +1 enhancement bonus to AC.</p></li><li><p>You gain resistance 2 against a single energy type (acid, cold, electricity, fire, or sonic).</p></li><li><p>Your reach is extended by 5 feet.</p></li></ul><p><span><em>Toggle on/off the selected effect in the Modifiers tab.</em></span><span style=\"box-sizing: border-box; user-select: text;\"><span><em>reach extension option must be manually tracked.</em></span></span></p>"
```

Comment:
Because we don't track reach is the exact reason for the comment on it. This item doesn't have all these features on at once, and the player instead selects the ones they want on. Thus the player needs to note if the reach option is active.